File: cpp/wejsciewyjscie.cpp
/*
 * szablon.cpp
 */


#include <iostream>

using namespace std;

int main(int argc, char **argv)
{
	float a, b ;   // deklaracja zmiennej
    a = b = 0;   // inicjacja zmiennej

    cout << "Podaj liczbę 1: ";
    cin >> a;
    cout << a << endl;

    cout << "Podaj liczbę 2: ";
    cin >> b;
    cout << b << endl;

    cout << "Suma: " << a + b << endl;
    cout << "Różnica: " << a - b << endl;
    cout << "Iloczyn: " << a * b << endl;
    cout << "Iloraz: " << a / b << endl;
		


	return 0;
}

File: python_inf/wejscie_wyjscie.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-
#
#  szablon.py

def dodaj():
    print(a +b)

def main(args):
    # a = 0  # inicjacja zmiennej
    a = int(input('Podaj liczbę 1: ')) 
    print(a)
    b = int(input('Podaj liczbę 2: ')) 
    print(b)
    
    dodaj()
    
    print("Suma: ", a + b)
    print("Różnica: ", a - b)
    print("Iloczyn: ", a * b)
    print("Iloraz: ", a / b)
    
    
    return 0
    
    # duck typing

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))
